asciidoc/modules/con_mg_changes-in-eventbus-javascript-client.adoc

```diff
@@ -0,0 +1,47 @@
+[id="changes-in-eventbus-javascript-client_{context}"]
+= Changes in EventBus JavaScript client
```

Comment:
@pmlopes Can you please review this file? I have rephrased the content. Can you check the terminology?